src/api/libcellml/variable.h

```diff
@@ -55,8 +55,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     UnitsPtr getUnits() const;
 
     /**
+     * @brief Set the initial value for this variable using a string.
+     * Set the initial value for this variable using a string.
+     *
+     * @overload
+     * @sa getInitialValue
+     * @param initialValue the initial value to set.
```

Comment:
Thanks @nickerso. Back to the current issue, I prefer when it's capitalised since there is no ":" between the variable name and its description (see the end of the page mentioned by @nickerso for an exemple of capitalisation).